sem_covid/entrypoints/etl_dags/finreg_worker.py

```diff
@@ -0,0 +1,385 @@
+    }
+    OPTIONAL {
+        ?work cdm:resource_legal_is_about_subject-matter ?subject_matter .
+        OPTIONAL {
+            ?subject_matter skos:prefLabel ?subject_matter_label .
+            FILTER (lang(?subject_matter_label)="en")
+        }
+    }
+    OPTIONAL {
+        ?work cdm:resource_legal_is_about_concept_directory-code ?directory_code .
+        OPTIONAL {
+            ?directory_code skos:prefLabel ?directory_code_label .
+            FILTER (lang(?directory_code_label)="en")
+        }
+    }
+    OPTIONAL {
+        ?work cdm:work_is_about_concept_eurovoc ?eurovoc_concept .
+        OPTIONAL {
+            ?eurovoc_concept skos:prefLabel ?eurovoc_concept_label .
+            FILTER (lang(?eurovoc_concept_label)="en")
+        }
+    }
+
+    OPTIONAL {
+        ?work cdm:work_created_by_agent ?author .
+        OPTIONAL {
+            ?author skos:prefLabel ?author_label .
+            FILTER (lang(?author_label)="en")
+        }
+    }
+    OPTIONAL {
+        ?work cdm:resource_legal_in-force ?in_force .
+    }
+    OPTIONAL {
+        ?work cdm:resource_legal_id_sector ?oj_sector .
+    }
+    OPTIONAL {
+        ?work cdm:resource_legal_published_in_official-journal ?full_oj .
+    }
+    OPTIONAL {
+        ?work cdm:resource_legal_comment_internal ?internal_comment .
+    }
+    OPTIONAL {
+        ?work cdm:work_date_document ?date_document .
+    }
+    OPTIONAL {
+        ?work cdm:work_date_creation ?date_created .
+    }
+    OPTIONAL {
+        ?work cdm:resource_legal_date_signature ?legal_date_signature .
+    }
+    OPTIONAL {
+        ?work cdm:resource_legal_date_entry-into-force ?legal_date_entry_into_force .
+    }
+
+    OPTIONAL {
+        ?work cdm:resource_legal_id_celex ?celex .
+    }
+    OPTIONAL {
+        ?work cdm:resource_legal_eli ?legal_eli .
+    }
+    OPTIONAL {
+        ?work cdm:work_id_document ?id_document .
+    }
+    OPTIONAL {
+        ?work owl:sameAs ?same_as_uri .
+    }
+
+    OPTIONAL {
+        ?exp cdm:expression_belongs_to_work ?work ;
+            cdm:expression_uses_language ?expr_lang ;
+            cdm:expression_title ?_title .
+
+        OPTIONAL {
+            ?manif_pdf cdm:manifestation_manifests_expression ?exp ;
+                cdm:manifestation_type ?type_pdf .
+            FILTER (str(?type_pdf) in ('pdf', 'pdfa1a', 'pdfa2a', 'pdfa1b', 'pdfx'))
+        }
+        OPTIONAL {
+            ?manif_html cdm:manifestation_manifests_expression ?exp ;
+                cdm:manifestation_type ?type_html .
+            FILTER (str(?type_html) in ('html', 'xhtml'))
+        }
+    }
+    BIND(IRI(concat(?manif_pdf,"/zip")) as ?pdf_to_download)
+    BIND(IRI(concat(?manif_html,"/zip")) as ?html_to_download)
+}
+GROUP BY ?work'''
+
+
+def make_request(query):
+    wrapper = SPARQLWrapper(config.EU_CELLAR_SPARQL_URL)
+    wrapper.setQuery(query)
+    wrapper.setReturnFormat(JSON)
+
+    return wrapper.query().convert()
+
+
+def get_single_item(query, json_file_name):
```

Comment:
I guess, this is can be renamed as well as it queries cellar and stores results into minio.